src/schema/v2/conversation/index.ts

```diff
@@ -322,7 +322,10 @@ export const ConversationType = new GraphQLObjectType<any, ResolverContext>({
           })
 
           return {
-            collectorProfile: data.collector_profile,
+            collectorProfile: {
+              ...data.collector_profile,
+              partnerId: to_id,
```

Comment:
**question:** 

1. Why is the partner ID included in this field?
2. On line 309 above there is a comment that says "Do not use this field for Partners." Do you know what that is referring to?

Reply:
1. I will add a comment, we need to pass down to the `CollectorProfile` type the `partnerId`, if we remove the need for Gravity to pass the `partner_id` that won't be necessary anymore, maybe it's better going into this direction specially for the Schema organization. I will create a follow-up ticket because that needs to be done on Gravity and then back to MP again.

2. I think the message is wrong, it should be `Do not use this field for Collectors`, can correct it in a sec 👍